saas/api/serializers.py

```diff
@@ -597,15 +599,39 @@ class AccessibleSerializer(serializers.ModelSerializer):
     printable_name = serializers.CharField(source='organization.printable_name')
     email = serializers.CharField(source='organization.email')
     role_description = RoleDescriptionSerializer(read_only=True)
+    home_url = serializers.SerializerMethodField()
+    settings_url = serializers.SerializerMethodField()
 
     class Meta:
         model = get_role_model()
         fields = ('created_at', 'request_key', 'grant_key',
+            'home_url', 'settings_url',
             'slug', 'printable_name', 'email', # Organization
             'role_description')                # RoleDescription
         read_only_fields = ('created_at', 'request_key', 'grant_key',
             'printable_name')
 
+    def get_settings_url(self, obj):
+        org = obj.organization
+        abs_uri = self.context['request'].build_absolute_uri
```

Comment:
Maybe though it seems incorrect if we look at the code in legal.py dealing with the same issue of getting an absolute uri.
```
        if settings.BUILD_ABSOLUTE_URI_CALLABLE:
            build_absolute_uri = import_string(
                settings.BUILD_ABSOLUTE_URI_CALLABLE)
            context.update({'site_url': build_absolute_uri(request)})
```

Reply:
So what should I do here?